app/src/pages/inside/projectSettingsPageContainer/content/notifications/footer/footer.jsx

```diff
@@ -66,9 +67,24 @@ export const NotificationsFooter = () => {
     },
   ];
 
+  if (isReadOnly) {
+    footerItems.unshift({
+      title: formatMessage(messages.unableToConfigure),
+      mainIcon: discoverPluginsIcon,
+      link: docsReferences.projectConfigurationDocs,
+      description: formatMessage(messages.unableToConfigureDescription),
+      openIcon: openInNewTabIcon,
+      event: PROJECT_SETTINGS_NOTIFICATIONS_EVENTS.CLICK_PROJECT_CONFIGURATION_LINK,
+    });
+  }
+
   return (
     <div className={cx('footer')}>
       <HelpPanel items={footerItems} />
     </div>
   );
 };
+
+NotificationsFooter.propTypes = {
+  isReadOnly: PropTypes.bool,
```

Comment:
I you define `propTypes` without required properties, the `defaultProps` should also be defined.